packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.js

```diff
@@ -140,9 +138,9 @@ ClipboardCopy.propTypes = {
   /** Tooltip message to display when clicking the copy button */
   clickTip: PropTypes.string,
   /** Custom flag to show that the input requires an associated id or aria-label. */
-  'aria-label': PropTypes.string,
+  textAriaLabel: PropTypes.string,
```

Comment:
Isn't this technically a breaking change? How many people are using `aria-label` or `toggle-aria-label` on ClipboardCopy out in the wild?

Reply:
I believe we are at a point where it is okay to introduce the breaking changes? Is that correct @tlabaj?

Reply:
This component is new and has not been released yet.  Since we are pre release the change is ok.